### ISSUE TITLE
decide fate of `dokku_git_sync` module

### ISSUE THREAD

**Issue by ltalirz** [dokku/ansible-dokku#113]
The `dokku git:sync` command became part of dokku as of version 0.23.0.

The `dokku_clone` module of the ansible role was changed to use `dokku git:sync` under the hood already.
The most straightforward thing would probably be to keep both `dokku_clone` and `dokku_git_sync` and let them reuse the same python code under the hood.

**Comment by josegonzalez** [comment]
I feel like we should deprecate one and only support the official method? And eventually delete the old one.

**Comment by josegonzalez** [comment]
We should just delete it. The `git-sync` plugin isn't even available to anyone except me as the source is private.

**Comment by ltalirz** [comment]
Ok, just to confirm: you propose to delete the `dokku_git_sync` ansible module, keeping only `dokku_clone`? Or the other way around?

Just saying that given that the underlying dokku command is `dokku git:sync`, `dokku_git_sync` seems to make sense as a name.
On the other hand, `dokku_clone` is probably what most users of the role were using since it did not require a commercial plugin.

**Comment by josegonzalez** [comment]
Yeah delete `dokku_git_sync`. In theory, that isn't going to break workflows for anyone, as they were already using `dokku_clone`.

**Comment by josegonzalez** [comment]
Okay I think maybe we should just rename to `dokku_git_sync`. That will make most sense to users looking at the official Dokku commands.

**Comment by ltalirz** [comment]
Agreed!

Let's deprecate `dokku_clone` in favor of `dokku_git_sync`, and, after a significant deprecation period (say, 12 months - I assume many people are using `dokku_clone`), we can delete the `dokku_clone` module.
